vendor/bat-native-ledger/src/bat/ledger/internal/database/database_contribution_info.cc

```diff
@@ -166,6 +167,7 @@ void DatabaseContributionInfo::OnGetRecord(
   info->retry_count = GetIntColumn(record, 4);
   info->processor =
       static_cast<type::ContributionProcessor>(GetIntColumn(record, 5));
+  info->created_at = GetInt64Column(record, 6);
```

Comment:
This field was not being populated, eventually resulting in overwriting the `created_at` value while passing through the AC flow.